semgrep/semgrep/target_manager.py

```diff
@@ -0,0 +1,234 @@
+import subprocess
+from pathlib import Path
+from typing import Dict
+from typing import List
+from typing import Set
+
+from semgrep.error import NotGitProjectError
+from semgrep.error import UnknownLanguageError
+from semgrep.util import partition_set
+
+
+def lang_to_exts(language: str) -> List[str]:
+    """
+        Convert language to expected file extensions
+
+        If language is not a supported semgrep language then
+        raises UnknownLanguageError
+    """
+    if language in ["python", "python2", "python3", "py"]:
```

Comment:
is this logic duplicated in ocaml or hopefully this is now the single source of truth?

Reply:
It still is for now. We can pull out the things in a follow up PR